ArduPlane/commands_logic.cpp

```diff
@@ -718,10 +714,15 @@ bool Plane::verify_loiter_to_alt()
     update_loiter(mission.get_current_nav_cmd().p1);
 
     //has target altitude been reached?
-    if (condition_value != 0) {
+    // condition_value == 1 means alt is ignored: we've either reached it or it's not valid
+    if (condition_value == 0) {
```

Comment:
Condition value 0 meant we hit the target altitude. We never had the concept of loiter to alt where alt was ignored. It is a nonsensical comment. The edge case handling for 0 was just to allow loiters to 0 meters relative (or darned close).